suite-native/intl/src/en.ts

```diff
@@ -175,6 +175,18 @@ export const en = {
                 confirm: 'Continue with <type></type>',
             },
         },
+        coinDiscoveryRunningScreen: {
+            title: 'Checking {coin} for balances & transactions.',
+            subtitle: 'This should take just a moment.',
+        },
+        coinDiscoveryFinishedScreen: {
+            title: {
+                singular: 'We’ve found <count>1</count> <coin>Bitcoin</coin> account',
+                plural: 'We’ve found <count>2</count> <coin>Bitcoin</coin> accounts',
```

Comment:
Format.js has rules for linting plural strings. What about trying to use that? Disclaimer: I never used that yet.

https://formatjs.io/docs/core-concepts/icu-syntax#plural-format

Reply:
I have tried to use it, but didnt succeed.